llvm/test/Transforms/SimplifyCFG/merge-calls-memprof.ll

```diff
@@ -1,5 +1,3 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Why did this get dropped?

Reply:
Teresa mentioned that when this test was introduced in https://github.com/llvm/llvm-project/pull/121359 it was copied over and not actually created using update_test_checks.py. Since I didn't use the tool to generate the checks for the new tests I dropped it.

Reply:
Why is it not possible to use generated checks in this test?

Reply:
It's probably possible to do so after splitting out the additional test cases I added into their own files. Is it a requirement for SimplifyCFG tests?

Reply:
> Teresa mentioned that when this test was introduced in #121359 it was copied over and not actually created using update_test_checks.py. Since I didn't use the tool to generate the checks for the new tests I dropped it.

IIRC I copied the test from somewhere else so did run update_test_checks.py as in the original test. Does it not work to use that for the new tests?

Reply:
Thanks for clarifying Teresa, I misunderstood your comment earlier. I've reverted the changes to the original test and added two new ones. The new tests also use update_test_checks to generate CHECK prefixes.

Ptal, @nikic